aiida/orm/utils/builders/code.py

```diff
@@ -195,7 +194,7 @@ def validate_installed(self):
         if messages:
             raise self.CodeValidationError(f'{messages}')
 
-    class CodeValidationError(Exception):
+    class CodeValidationError(ValueError):
```

Comment:
I would maybe add a small comment on the commit message mentioning this change (as it doesn't seem immediately related to the others).

Reply:
Updated

Reply:
Ok, great! I meant it more like to take it into consideration for the final marge, but good to already have the original commit as a reference =)